services/version-api/src/lambdas/getStatus.ts

```diff
@@ -0,0 +1,66 @@
+import * as response from '../libs/response';
+import config from '../libs/config';
+import params from '../libs/params';
+import log from '../libs/logs';
+
+import { getApplicationUpdateStatus } from '../helpers/getApplicationUpdateStatus';
+
+const USER_AGENT = 'User-Agent';
+const SPLIT_SYMBOL = '/';
+const SUPPORTED_OS = ['ios', 'android'];
```

Comment:
Put consts after types and interface declarations?

Reply:
Yes, constants should be after interfaces and types